crates/cargo-codspeed/src/app.rs

```diff
@@ -16,15 +16,30 @@ struct Cli {
     command: Commands,
 }
 
+/// Package selection flags
+#[derive(Args)]
+struct PackageSelection {
+    /// Select all packages in the workspace
+    #[arg(long)]
+    workspace: bool,
+    /// Exclude packages
+    #[arg(long)]
+    exclude: Vec<String>,
+    /// Package to select
+    #[arg(short, long)]
+    package: Vec<String>,
+}
```

Comment:
If any of these is not specified, will it work?
ie, will we have `workspace == false`, `exclude == []` and `package == []`, or will it error?

Reply:
the behavior without specifying anything is tested and it matches the definition from cargo